gazelle/generate.go

```diff
@@ -197,7 +224,7 @@ func (lang *JS) GenerateRules(args language.GenerateArgs) language.GenerateResul
 	}
 }
 
-func (lang *JS) collectSources(args language.GenerateArgs, jsConfig *JsConfig) ([]string, []string, []string, map[string]bool, bool, bool) {
+func (lang *JS) collectSources(args language.GenerateArgs, jsConfig *JsConfig) ([]string, []string, []string, map[string]bool, bool) {
```

Comment:
we should consider returning a type/struct, getting harder to understand what is returned